app/assets/javascripts/discourse/widgets/default-notification-item.js.es6

```diff
@@ -154,6 +155,17 @@ export const DefaultNotificationItem = createWidget(
           );
         }
       });
+    },
+
+    mouseUp(e) {
+      // dismiss notification on middle click
+      if (e.which === 2 && !this.attrs.read) {
+        this.attrs.set("read", true);
+        ajax("/notifications/mark-read", {
```

Comment:
Shouldn't we do that in https://github.com/discourse/discourse/blob/de81bf76dc249dd5723d2ab85f87c08da40a0016/app/assets/javascripts/discourse/widgets/default-notification-item.js.es6#L139-L141 instead of in a different event?

Reply:
The `click` event handler is called for left click only, so I had to use `mouseUp` which is triggered on both left and middle clicks.